### PR TITLE
AMBR-977: Implement updated Figshare widget

### PR DESCRIPTION
**Pull request by msingh05**

### CODE REVIEW

**Comment by sgatchell** [comment]
@msingh05 how do I test this?

File: src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js [hunk 1]
```diff
@@ -47,4 +71,28 @@ if (window.figshare) {
     window.loadedWidgets = loadedWidgets;
 
   });
+
+
+  // doubleclickad gets added to <aside> element of this widget. so rename it to <div>
```
Comment (sgatchell) [inline src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js]: I want to understand this problem better.  What's an example article that I can use to see the issue?

**Comment by msingh05** [comment]
> @msingh05 how do I test this?

Figshare widget should work as same, there are some minor visual changes.

File: src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js (hunk 1 above)
Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js]: you can reproduce the issue with any article which has supporting information. Comment the function call and you can see the issue. I am attaching the screenshot also.

Comment (msingh05) [reply src/main/webapp/WEB-INF/themes/root/resource/js/components/figshare.js]: Screen Shot 2020-02-19 at 3 05 11 PM

**Comment by sgatchell** [comment]
I observed a bug in the new widget.  The bottom description does not change when you select a different file.  See how it's showing the description of the the first file even though I've scrolled to 3/3.  The dev environment does not have this bug - it updates the description to match the selected file.
Screenshot from 2020-02-20 08-52-54

**Comment by msingh05** [comment]
@sgatchell can you check again. marketing has fixed their code so I removed the hack.

**Pull request merged**